distributed/scheduler.py

```diff
@@ -370,54 +481,62 @@ def status(self, new_status):
             raise TypeError(f"expected Status or str, got {new_status}")
 
     @property
-    def host(self):
-        return get_address_host(self.address)
+    def time_delay(self):
+        return self._time_delay
+
+    @property
+    def used_resources(self):
+        return self._used_resources
+
+    @property
+    def versions(self):
+        return self._versions
 
     def clean(self):
         """ Return a version of this object that is appropriate for serialization """
-        ws = WorkerState(
-            address=self.address,
-            pid=self.pid,
-            name=self.name,
-            nthreads=self.nthreads,
-            memory_limit=self.memory_limit,
-            local_directory=self.local_directory,
-            services=self.services,
-            nanny=self.nanny,
-            extra=self.extra,
+        ws: WorkerState = WorkerState(
+            address=self._address,
+            pid=self._pid,
+            name=self._name,
+            nthreads=self._nthreads,
+            memory_limit=self._memory_limit,
+            local_directory=self._local_directory,
+            services=self._services,
+            nanny=self._nanny,
+            extra=self._extra,
         )
-        ws.processing = {ts.key for ts in self.processing}
+        ws._processing = {ts.key for ts in self._processing}
```

Comment:
This gives us an error now as this is typed as `dict` based on the docstring above and the default value, which is also a `dict`. However here we are assigning it a `set`. Submitted PR ( https://github.com/dask/distributed/pull/4295 ) to provide one way to correct this. Though I don't know that it is the right solution necessarily.

Reply:
Merged.  Thanks

Reply:
Thanks Matt! 😄